plugins/powerd/fu-plugin-powerd.c

```diff
@@ -86,6 +86,10 @@ fu_plugin_update_prepare (FuPlugin *plugin,
 		return FALSE;
 	}
 
+	/* permit updates when the device does not care about power conditions */
```

Comment:
This should be done before calling on dbus. We don't care about what powerd says if the --ignore-power is set